README.md

```diff
@@ -173,6 +173,17 @@ You can now head to (if pushpin is correctly configured and use default port 799
 
 :warning: Using tracim without pushpin will mean no live message leading to unrefreshed frontend information.
 
+### Upkeep
+
+If you use default "file" storage for session, new session file will be created but not removed, which mean lot of unneeded file.
+To mitigate this, you could remove file older than 10 day with :
+
+    find . -type f -mtime +10 -print -exec rm {} \;
+
+You should use this command in both session data and session lock dirs.
+
+Another way to solve this issue is to set session to use another backend(we do support all beaker backend),
+see [session](/backend/doc/setting.md) documentation for more information.
```

Comment:
For me, its better to indicate that its necessary to match day value by lifetime token. (and set value to default value of lifetime token. Example: if lifetime is 7days, day value can be 8 days).

Reply:
changed to 8 days.

Reply:
thanks